libs/application/ui-fields/src/lib/DividerFormField/DividerFormField.tsx

```diff
@@ -13,9 +13,10 @@ export const DividerFormField: FC<
   }>
 > = ({ field, application }) => {
   const { formatMessage, lang: locale } = useLocale()
+  const { marginTop, marginBottom } = field
```

Comment:
How about
` const { marginTop = 5, marginBottom = 1 } = field`
To keep the value passing below a bit cleaner?

Reply:
I'm thinking I'm gonna leave this one specifically, because if we destructure here, then the values will also default to 5 and 1 for the non-title type of divider, which is not "backwards compatible" since they used to always have 0 margin.